### ISSUE TITLE
No Obvious  "Developer Instructions" / "Get Involved Page"

### ISSUE THREAD

**Issue by slacka** [Antergos/Cnchi#687]
There's lots of help wanted /PR requests in the issues pages, no obvious information on how to get started. 

The Antergos home page 'Development' only links to "code". There are two wikis(antergos.com and github), but neither of them has a "Getting Started" section for Developers. At a minimum there should be a sections on:

- How to build Cnchi
- Recommended Testing Environment or Procedure
- Development Tips
 
 For example, I wanted to try to work on issue#686 but I have no idea where to begin. 

It doesn't have to be beautiful. Just a dump of commands and procedure that the devs do without even thinking about it. In other words, scroll through your `./bash_history` and give us some notes along the way.

**Comment by stratus-ss** [comment]
Yes I agree with this statement. It can be tough to jump in sometimes. Especially without knowing what the official way to solve a problem is.
